package.json

```
@@ -50,11 +55,15 @@
       "name": "Alan Gutierrez",
       "email": "alan@prettyrobots.com",
       "web": "http://www.prettyrobots.com/"
+    },
+    {
+      "name": "Eric Newport",
+      "email": "kethinov@gmail.com",
+      "web": "https://github.com/kethinov"
     }
   ],
   "bugs": {
-    "email": "jindw@xidea.org",
-    "url": "http://github.com/xmldom/xmldom/issues"
+    "url": "https://github.com/xmldom/xmldom/issues"
   },
   "license": "(LGPL-2.0 or MIT)"
```

can you fix the license field, too? To stop npm bugging about it, it needs to be:
```suggestion
   "license": "(LGPL-2.0 OR MIT)"
```

(capitalized OR)

---

Depends on how we want to handle this https://github.com/xmldom/xmldom/pull/11

---

I would vote to close PR #11 and commit the suggestion by @timbru31.